providers/aws/services/appstream/appstream_fleet_maximum_session_duration/appstream_fleet_maximum_session_duration.py

```diff
@@ -0,0 +1,28 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.appstream.appstream_client import appstream_client
+
+max_session_duration_seconds = 10 * 60 * 60
+"""max_session_duration_seconds 36000"""
```

Comment:
How about setting this variable in config.yaml and get it with get_config_var function?

Reply:
Done!